src/index.html

```diff
@@ -11,12 +11,53 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="img-wraper">
+        <img
+          src="./images/logo.png"
+          alt="Logotype"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li class="is-active">
+            <a href="#">Apple</a>
+          </li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li>
+            <a
+              data-qa="hover"
+              href="#"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li><a href="#">Gadgets</a></li>
+          <li><a href="#">Tablets</a></li>
+          <li><a href="#">Photo</a></li>
+          <li><a href="#">Video</a></li>
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol. It's better to use meaningful names, like `#apple`, `#samsung`, etc. This will improve your page's SEO.